single_cell_RNAseq/bin/filter_barcodes_pre_fmx.R

```diff
@@ -52,11 +52,18 @@ if (adt.present){
 
 # plots with final reviewed params
 params = load_params(params_df) 
-plot_list = make_plots(sobj, params)
-num_rows = ifelse(adt.present, 3, 2)
-pdf( sprintf("%s_diagnostic_plots_final.pdf", LIBRARY) , width=25, height=7*num_rows)
-ggarrange(plotlist=plot_list, ncol=3,nrow=num_rows)
-dev.off()
+plot_list = suppressWarnings(make_plots(sobj, params, adt.present))
+num_rows = ifelse(adt.present, 5, 3)
+merge = ggarrange(plotlist=plot_list, ncol=4,nrow=num_rows)
+ggsave(merge, file=sprintf("%s_diagnostic_plots_reviewed.png", LIBRARY) , width=30, height=7*num_rows, bg="white", dpi=72)
+
+if ("cellranger_cell" %in% colnames(sobj@meta.data)){
+  plot_list = suppressWarnings(make_plots(sobj, params, adt.present, group="cellranger_cell"))
+  num_rows = ifelse(adt.present, 5, 3)
+  merge = ggarrange(plotlist=plot_list, ncol=4,nrow=num_rows)
+  ggsave(merge, file=sprintf("%s_cr_diagnostic_plots_reviewed.png", LIBRARY) , width=30, height=7*num_rows, bg="white", dpi=72)
+}
```

Comment:
Super tiny nit here, so feel free to ignore if the work you do to test makes it feel not worth it to just save a few lines of code...

Seems like you could start this chunk with 
```
if ("cellranger_cell" %in% colnames(sobj@meta.data)) {
    plot_list = suppressWarnings(make_plots(sobj, params, adt.present, group="cellranger_cell"))
} else {
    plot_list = suppressWarnings(make_plots(sobj, params, adt.present))
}
```
and then remove the duplication of the num_rows, merge, and ggsave lines which all look the same.